build_tools/third_party/swiftshader/build_vk_swiftshader.sh

```diff
@@ -33,6 +33,9 @@ set -e
 # See https://vulkan.lunarg.com/doc/view/1.1.70.1/windows/loader_and_layer_interface.html
 # for further details about the Vulkan loader and ICDs.
 
+SWIFTSHADER_COMMIT=6287c18b1d249152563f0cb2d5cb0c6d0eb9e3d6
+SWIFTSHADER_DIR=/tmp/swiftshader_install
+
 # Check that we're in the project root so our relative paths work as expected.
```

Comment:
I think no longer necessary. If it is though, I'd just `cd "$(git rev-parse --show-toplevel)"`

Reply:
Good idea!